recipes/jaws/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.2.3" %}
+
+package:
+  name: jaws
+  version: {{ version }}
+
+source:
+  url: https://github.com/jaws/jaws/archive/{{ version }}.tar.gz
+  sha256: b8cd6e76f01a3786d32880c757b02aa82651eb9fffeb0bc8aa155dac02212b38
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.11
+  run:
+    - python
+    - pandas
+    - numpy
+    - xarray
+    - pytz
+    - netcdf4
+    - matplotlib
+    - netcdf4
+
+test:
+  imports:
+    - unittest
+    - tempfile
+
+about:
+  home: https://github.com/jaws/jaws
+  license: Apache, Version 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Software to convert idiosyncratic ASCII formats to netCDF formats.'
+
+extra:
+  recipe-maintainers:
+    - czender
```

Comment:
@czender are you ok with being a maintainer?

Reply:
i'm OK with being a fallback maintainer but @ajcse1 should be THE default maintainer

Reply:
If you are going to be a fallback maintainer you most likely should list yourself on the recipe (otherwise you won't have merge rights on the feedstock).